ServiceBoxWeb/src/main/resources/templates/employee/service-order-list.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<html lang="pl" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>ServiceBox</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+    <link th:href="@{/css/style.css}" rel="stylesheet" type="text/css">
+</head>
+<body>
+<div class="container">
+    <div class="row">
+        <div class="col-md-12">
+            </br>
```

Comment:
Nie powinno się wykorzystywać znacznika `<br>` do robienia odstępów - lepiej dodać margines górny do tabeli korzystając z CSS `margin-top: 30px`, lub w tym przypadku z klasy Bootstrapa `mt-5`.
HTML odpowiada za treść strony, jej semantykę, natomiast to jak coś ma wyglądać powinniśmy definiować z wykorzystaniem CSS :)

https://getbootstrap.com/docs/5.2/utilities/spacing/#margin-and-padding

Reply:
Poprawiono: <div class="col-md-12 d-flex mt-5">